shell/components/Resource/Detail/Metadata/IdentifyingInformation/composable.ts

```diff
@@ -1,35 +1,58 @@
-import { useI18n } from '@shell/composables/useI18n';
-import { computed, ComputedRef, markRaw, toValue } from 'vue';
-import LiveDate from '@shell/components/formatter/LiveDate.vue';
-import { useStore } from 'vuex';
+import { computed, ComputedRef, toValue } from 'vue';
 import { Row } from '@shell/components/Resource/Detail/Metadata/IdentifyingInformation/index.vue';
+import {
+  useCertificate,
+  useExpires,
+  useImage, useIssuer, useLiveDate, useNamespace, useReady, useSecretType,
+  useServiceAccount
+} from '@shell/components/Resource/Detail/Metadata/IdentifyingInformation/identifying-fields';
+import { useStore } from 'vuex';
+import { useI18n } from '@shell/composables/useI18n';
 
-export const useLiveDate = (resource: any): ComputedRef<Row> => {
-  const store = useStore();
-  const i18n = useI18n(store);
-  const resourceValue = toValue(resource);
+export const useDefaultIdentifyingInformation = (resource: any): ComputedRef<Row[]> => {
+  return computed(() => {
+    const namespace = useNamespace(resource);
+    const liveDate = useLiveDate(resource);
 
-  return computed(() => ({
-    label:         i18n.t('component.resource.detail.metadata.identifyingInformation.age'),
-    valueOverride: {
-      component: markRaw(LiveDate),
-      props:     { value: resourceValue.creationTimestamp }
-    },
-    value: resourceValue.age,
-  }));
+    return [
+      namespace.value,
+      liveDate.value
+    ];
+  });
 };
 
-export const useDefaultIdentifyingInformation = (resource: any): ComputedRef<Row[]> => {
+export const useSecretIdentifyingInformation = (resource: any): ComputedRef<Row[]> => {
+  return computed(() => {
+    const rows = [
+      useSecretType(resource),
+      useServiceAccount(resource),
+      useCertificate(resource),
+      useIssuer(resource),
+      useExpires(resource),
+    ];
+
+    return rows
+      .filter((r) => typeof r !== 'undefined')
+      .map((r) => r.value);
+  });
+};
+
+export const useDefaultWorkloadIdentifyingInformation = (resource: any): ComputedRef<Row[]> => {
```

Comment:
I just noticed this slipped in. This isn't needed for this PR but if you don't mind it here it's relevant for a future pr and I'd just like to leave it.